src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -1948,3 +1948,10 @@ def test_cloudpickle_fast(pyi_builder):
         modname = "cloudpickle.cloudpickle_fast"
         mod = importlib.import_module(modname)
     """)
+
+
+# Check if pptx template is included
+@importorskip('pptx')
+def test_pptx(pyi_builder):
+    import pptx
+    pptx.Presentation()
```

Comment:
This should be 
```python
# Check if pptx template is included
@importorskip('pptx')
def test_pptx(pyi_builder):
    pyi_builder.test_source("""
        import pptx
        pptx.Presentation()
    """)
```

Otherwise, you are not testing the PyInstaller-frozen version of the program.